### PR TITLE
updated server limits

### PR DESCRIPTION
**Pull request by juliaElastic**
## What is the problem this PR solves?

Unblocking scalability testing.

## How does this PR solve the problem?

The current server limits are causing scalability testing to be blocked, because the containers created for 50k, 75k, etc. agents are too small.

Took the recommended Integration Server memory sizes from here https://docs.google.com/spreadsheets/d/1V3RlxZOkNQYpAMPGtWUtLRggaHYqVIISQacSynyVDkw/edit#gid=1443003738

EDIT: Took the limits for 50k, 75k and max limit from here (except for memory): https://github.com/elastic/observability-perf/issues/208#issuecomment-1263675432

Open questions:
- Are these recommendations correct? 16GB for 15k and 25k agents, 24GB for 50k and 75k, 32GB for 100k.
- Do any of the other settings have to be updated (e.g. `cache_limits`, `checkin_limits`)? Now I copied the `gt125000` file for the new files and only modified the `recommended_min_ram`.

## Checklist

- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] I have made corresponding change to the default configuration files
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have added an entry in `CHANGELOG.next.asciidoc` or `CHANGELOG-developer.next.asciidoc`.

## Related issues

Closes https://github.com/elastic/fleet-server/issues/1857

### CODE REVIEW

File: CHANGELOG.next.asciidoc
Comment (michel-laterman) [inline CHANGELOG.next.asciidoc:31]: (nitpick) can you also reference the issue?

**Review by michel-laterman — APPROVED** [review]
lgtm.

btw how did we arrive at these numbers?

**Comment by juliaElastic** [comment]
> lgtm.
> 
> btw how did we arrive at these numbers?

See the links in the issue, the numbers might change, we have a meeting today about it.

**Review by AndersonQ — APPROVED** [review]

**Review by michalpristas — APPROVED** [review]

**Comment by juliaElastic** [comment]
Latest update:

| recommended_min_ram | num_agents |
| --- | --- |
|8192|10000-29999|
|16384|30000-49999|
|32768|50000-99999|
|65536|10000+|

**Comment by juliaElastic** [comment]
As discussed with @joshdover and @pjbertels this pr is on hold until the new limits are tested and confirmed.

**Comment by jlind23** [comment]
@joshdover @ablnk just updated the gt50000 with the settings used by Andrei and resolved the conflicts. Could you please double check?

@juliaElastic could you look at it and merge afterwards please?

**Comment by joshdover** [comment]
After discussion earlier, we should not merge this yet.

**Comment by jlind23** [comment]
Closing this as discussed in the related issue.

**Pull request closed without merging**
